rnd/autogpt_server/autogpt_server/blocks/http.py

```diff
@@ -0,0 +1,50 @@
+import requests
```

Comment:
@Bentlybro I repurposed the discordsendmessage block to the generic httprequest block for now to make the build green and use it for more generic purpose.

Feel free to revert an reintroduce this again with a fixed test if you think using the generic block would still be complicated :)

Reply:
alright, it will be fine as a generic http block and ill just leave out the discord message webhook block